cmd/error.go

```diff
@@ -76,11 +77,22 @@ func fatalIf(err *probe.Error, msg string) {
 	console.Fatalln(fmt.Sprintf("%s %s", msg, err))
 }
 
+var errorIfExitStatus int
```

Comment:
It is a poor programming practice to declare global variables like this one. Global variables that are accessed across files are to be placed in globals.go. That way we know the ones that we have are must have.